chart/templates/gateway/envoy-ai-gateway.yaml

```diff
@@ -0,0 +1,107 @@
+{{- if .Values.envoyAIGateway.enabled -}}
```

Comment:
Let's not include these files in the helm chart I think, let's make them an example instead. Because most of them should be user-defined.